airbyte-integrations/connectors/source-okta/unit_tests/test_source.py

```diff
@@ -0,0 +1,122 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import MagicMock
+
+from airbyte_cdk.sources.streams.http.auth import TokenAuthenticator
+from source_okta.source import (
+    CustomRoles,
+    GroupMembers,
+    GroupRoleAssignments,
+    Groups,
+    Logs,
+    OktaOauth2Authenticator,
+    SourceOkta,
+    UserRoleAssignments,
+    Users,
+)
+
+
+class TestAuthentication:
+
+    def test_init_token_authentication_init(self, token_config, auth_token_config):
+        config = token_config
+        source_okta = SourceOkta()
+        token_authenticator_instance = source_okta.initialize_authenticator(config=config)
+        assert isinstance(token_authenticator_instance, TokenAuthenticator)
```

Comment:
No need resign test config to config variable.

```suggestion
        source_okta = SourceOkta()
        token_authenticator_instance = source_okta.initialize_authenticator(config=token_config)
        assert isinstance(token_authenticator_instance, TokenAuthenticator)
```

The same for other tests.

Reply:
done